tests/tasks/test_noref_beam.py

```diff
@@ -0,0 +1,400 @@
+import numpy as np
```

Comment:
Some/all (?) of these would be covered by the tests that will be part of https://github.com/sbi-benchmark/sbibm/pull/29. Maybe it's easier to remove the tests from this PR and merge this one and #29 together when they are ready?

Reply:
Yes, I need to remove that.

Reply:
done with a676e65